scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -44,7 +48,22 @@ namespace <%= namespace %>.SDK.iOS {
         }
 
         public void DidFinishCheckoutSession(string serializedMessage) {
-            //TODO
+            var message = NativeMessage.CreateFromJSON(serializedMessage);
+            var paymentStatus = (NativePaymentStatus) Enum.Parse(typeof(NativePaymentStatus), (string) message.Content);
```

Comment:
Yes that's correct. If we are sending anything else an exception will be thrown which I think makes sense since we don't want the SDK developer to assume that they can send anything and have it be handled without adding the handling code in C#.